contentful-export.json

```diff
@@ -15,7 +15,7 @@
         "updatedAt": "2021-05-29T18:03:18.122Z",
         "environment": {
           "sys": {
-            "id": "dev",
+            "id": "master",
```

Comment:
Small favor, can create an export file from your Contentful? I'm not sure if this file will work after changing it manually, so I'll prepare new one after importing your's.

Reply:
@dCzenczek I created a branch for this changes to check it locally. ;)

Reply:
@SikoraWojciech I exported file from my contentful and manually move changes regarding FAQ component, after changed it I created new project in contentful and than import from this file and everything was working correctly